shoes-package/lib/warbler/traits/shoes.rb

```diff
@@ -29,15 +29,37 @@ def self.requires?(trait)
         [Traits::Jar, Traits::NoGemspec].include? trait
       end
 
+      INIT_CONTENTS = <<~EOS
+        require 'shoes'
+        require 'shoes/swt'
+
+        Shoes::Swt.initialize_backend
+      EOS
+
       def after_configure
-        config.init_contents << StringIO.new("require 'shoes'\nrequire 'shoes/swt'\nShoes::Swt.initialize_backend\n")
+        config.init_contents << StringIO.new(INIT_CONTENTS)
       end
 
       def update_archive(jar)
         super
         add_main_rb(jar, apply_pathmaps(config, default_executable, :application))
```

Comment:
It comes from the [base `Warbler::Traits` module](https://github.com/jruby/warbler/blob/master/lib/warbler/traits.rb#L82-L87)